src/protobuf-net.MSBuild/GenerateProtoBufCode.cs

```diff
@@ -123,5 +133,13 @@ public override bool Execute()
 
             return true;
         }
+
+        private static void setIfNotNull(Dictionary<string, string> options, string key, string value)
```

Comment:
Naming: `SetIfNotNull` (capitalization) - might be clearer as `SetIfAssigned` or similar - see comment below on what it checks
Location: it looks like this would work well as a static local method, since it is only used from one method?